pkg/projects/Microsoft.NETCore.DotNetHost/win/Microsoft.NETCore.DotNetHostResolver.pkgproj

```diff
@@ -13,7 +13,7 @@
   
   <ItemGroup>
     <Dependency Include="Microsoft.NETCore.DotNetHost">
-      <Version>$(HostFullVersion)</Version>
```

Comment:
Why do we need this rename in servicing branch? I dont believe it is imperative for the fix. Please feel free to make it in master.